A01-Extensions/A01-How_to_write_ASDF_extensions.ipynb

```diff
@@ -0,0 +1,929 @@
+{
```

Comment:
<p>Astronomers are going to find the 3d designation confusing since they are thinking of 3d ellipsoids. Perhaps rotated_ellipses? or something like that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>